### ISSUE TITLE
podman_volume broken in 1.10.3

### ISSUE THREAD

**Issue by zyphermonkey** [containers/ansible-podman-collections#628]
**Is this a BUG REPORT or FEATURE REQUEST? (leave only one on its own line)**

/kind bug

**Description**

**Steps to reproduce the issue:**

1. upgrade collection from 1.10.2 to 1.10.3

2. run task
    ```yaml
    - name: Create the buffer var volume
      containers.podman.podman_volume:
        name: "{{ buffer_vol_name }}"
        state: present
    ```

**Describe the results you received:**

```shell
An exception occurred during task execution. To see the full traceback, use -vvv. The error was: SyntaxError: invalid syntax
fatal: [host01]: FAILED! => {"changed": false, "module_stderr": "", "module_stdout": "Traceback (most recent call last):
 File \"/home/owner/.ansible/tmp/ansible-tmp-1692437778.2696848-19847-248149379250372/AnsiballZ_podman_volume.py\", line 100, in <module>
   _ansiballz_main()
 File \"/home/owner/.ansible/tmp/ansible-tmp-1692437778.2696848-19847-248149379250372/AnsiballZ_podman_volume.py\", line 92, in _ansiballz_main
   invoke_module(zipped_mod, temp_path, ANSIBALLZ_PARAMS)
 File \"/home/owner/.ansible/tmp/ansible-tmp-1692437778.2696848-19847-248149379250372/AnsiballZ_podman_volume.py\", line 41, in invoke_module
   run_name='__main__', alter_sys=True)
 File \"/usr/lib64/python2.7/runpy.py\", line 176, in run_module
   fname, loader, pkg_name)
 File \"/usr/lib64/python2.7/runpy.py\", line 82, in _run_module_code
   mod_name, mod_fname, mod_loader, pkg_name)
 File \"/usr/lib64/python2.7/runpy.py\", line 72, in _run_code
   exec code in run_globals
 File \"/tmp/ansible_containers.podman.podman_volume_payload_nkIsdT/ansible_containers.podman.podman_volume_payload.zip/ansible_collections/containers/podman/plugins/modules/podman_volume.py\", line 106, in <module>
 File \"/tmp/ansible_containers.podman.podman_volume_payload_nkIsdT/ansible_containers.podman.podman_volume_payload.zip/ansible_collections/containers/podman/plugins/module_utils/podman/common.py\", line 165
   result['diff']['before'].update({f'systemd_{file_name}.service': ''})
                                                                 ^\r\nSyntaxError: invalid syntax\r\n", "msg": "MODULE FAILURE\nSee stdout/stderr for the exact error", "rc": 1}
```

**Describe the results you expected:**

Since this volume is already created on my hosts this should just return unchanged, this could also returned changed.

**Additional information you deem important (e.g. issue happens only occasionally):**

I believe this is related to changes made in https://github.com/containers/ansible-podman-collections/blame/fea8b255e59192c4a31acd177debbf6506632e5d/plugins/module_utils/podman/common.py#L106

Downgrading to 1.10.2 fixes the issue.

```yaml
---
collections:
  - name: containers.podman
    version: 1.10.2
```

```shell
ansible-galaxy collection install -r collections/requirements.yml
```

**Version of the `containers.podman` collection:**
**Either git commit if installed from git: `git show --summary`**
**Or version from `ansible-galaxy` if installed from galaxy: `ansible-galaxy collection list | grep containers.podman`**

```
containers.podman 1.10.3
```

**Output of `ansible --version`:**

```
ansible [core 2.11.12]
  config file = /home/vagrant/git/ansible-for-splunk/ansible.cfg
  configured module search path = ['/home/vagrant/.ansible/plugins/modules', '/usr/share/ansible/plugins/modules']
  ansible python module location = /usr/local/lib/python3.6/site-packages/ansible
  ansible collection location = /home/vagrant/.ansible/collections:/usr/share/ansible/collections
  executable location = /usr/local/bin/ansible
  python version = 3.6.8 (default, Nov 16 2020, 16:55:22) [GCC 4.8.5 20150623 (Red Hat 4.8.5-44)]
  jinja version = 3.0.3
  libyaml = True

```

**Output of `podman version`:**

```
Version:            1.6.4
RemoteAPI Version:  1
Go Version:         go1.12.12
OS/Arch:            linux/amd64
```

**Output of `podman info --debug`:**

``` yaml
debug:
  compiler: gc
  git commit: ""
  go version: go1.12.12
  podman version: 1.6.4
host:
  BuildahVersion: 1.11.7
  CgroupVersion: v1
  Conmon:
    package: conmon-2.0.15-1.el7_8.x86_64
    path: /usr/bin/conmon
    version: 'conmon version 2.0.15, commit: 372b4a12f1c2df4f70c280d41173b60acd3f1260'
  Distribution:
    distribution: '"rhel"'
    version: "7.9"
  MemFree: 1492135936
  MemTotal: 12428206080
  OCIRuntime:
    name: runc
    package: runc-1.0.0-69.rc10.el7_9.x86_64
    path: /usr/bin/runc
    version: 'runc version spec: 1.0.1-dev'
  SwapFree: 1974202368
  SwapTotal: 2147479552
  arch: amd64
  cpus: 8
  eventlogger: journald
  hostname: host01
  kernel: 3.10.0-1160.95.1.el7.x86_64
  os: linux
  rootless: false
  uptime: 237h 52m 15.05s (Approximately 9.88 days)
registries:
  blocked: null
  insecure: null
  search:
  - registry.access.redhat.com
  - registry.redhat.io
  - docker.io
store:
  ConfigFile: /etc/containers/storage.conf
  ContainerStore:
    number: 1
  GraphDriverName: overlay
  GraphOptions: {}
  GraphRoot: /var/lib/containers/storage
  GraphStatus:
    Backing Filesystem: xfs
    Native Overlay Diff: "true"
    Supports d_type: "true"
    Using metacopy: "false"
  ImageStore:
    number: 6
  RunRoot: /var/run/containers/storage
  VolumePath: /var/lib/containers/storage/volumes


```

**Package info (e.g. output of `rpm -q podman` or `apt list podman`):**

```
podman-1.6.4-36.el7_9.x86_64
```

**Playbok you run with ansible (e.g. content of `playbook.yaml`):**

``` yaml
- name: Create the buffer var volume
  containers.podman.podman_volume:
    name: "{{ buffer_vol_name }}"
    state: present
```

**Command line and output of ansible run with high verbosity**

**Please NOTE: if you submit a bug about idempotency, run the playbook with `--diff` option, like:**

`ansible-playbook -i inventory --diff -vv playbook.yml`

```
An exception occurred during task execution. To see the full traceback, use -vvv. The error was: SyntaxError: invalid syntax
fatal: [host01]: FAILED! => {"changed": false, "module_stderr": "", "module_stdout": "Traceback (most recent call last):
 File \"/home/owner/.ansible/tmp/ansible-tmp-1692437778.2696848-19847-248149379250372/AnsiballZ_podman_volume.py\", line 100, in <module>
   _ansiballz_main()
 File \"/home/owner/.ansible/tmp/ansible-tmp-1692437778.2696848-19847-248149379250372/AnsiballZ_podman_volume.py\", line 92, in _ansiballz_main
   invoke_module(zipped_mod, temp_path, ANSIBALLZ_PARAMS)
 File \"/home/owner/.ansible/tmp/ansible-tmp-1692437778.2696848-19847-248149379250372/AnsiballZ_podman_volume.py\", line 41, in invoke_module
   run_name='__main__', alter_sys=True)
 File \"/usr/lib64/python2.7/runpy.py\", line 176, in run_module
   fname, loader, pkg_name)
 File \"/usr/lib64/python2.7/runpy.py\", line 82, in _run_module_code
   mod_name, mod_fname, mod_loader, pkg_name)
 File \"/usr/lib64/python2.7/runpy.py\", line 72, in _run_code
   exec code in run_globals
 File \"/tmp/ansible_containers.podman.podman_volume_payload_nkIsdT/ansible_containers.podman.podman_volume_payload.zip/ansible_collections/containers/podman/plugins/modules/podman_volume.py\", line 106, in <module>
 File \"/tmp/ansible_containers.podman.podman_volume_payload_nkIsdT/ansible_containers.podman.podman_volume_payload.zip/ansible_collections/containers/podman/plugins/module_utils/podman/common.py\", line 165
   result['diff']['before'].update({f'systemd_{file_name}.service': ''})
                                                                 ^\r\nSyntaxError: invalid syntax\r\n", "msg": "MODULE FAILURE\nSee stdout/stderr for the exact error", "rc": 1}
```

**Additional environment details (AWS, VirtualBox, physical, etc.):**

**Comment by sshnaidm** [comment]
Python 2?

**Comment by zyphermonkey** [comment]
We're on rhel7 currently and python 2 is the only default python supported by the core OS.

**Comment by sshnaidm** [comment]
Fixed by #643
